draft-ietf-intarea-provisioning-domains.xml

```diff
@@ -750,19 +752,9 @@
         Whenever an unknown key is encountered, it MUST be ignored along with
         its associated elements.</t>
 
-        <section anchor="keys-private" title="Private Extensions">
-          <t>JSON keys starting with "x-" are reserved for private use and can
-          be utilized to provide information that is specific to vendor, user
-          or enterprise. It is RECOMMENDED to use one of the patterns
-          "x-FQDN-KEY" or "x-PEN-KEY" where FQDN is a fully qualified domain
-          name or PEN is a <xref target="PEN">private enterprise number</xref>
-          under control of the author of the extension to avoid
-          collisions.</t>
-        </section>
-
```

Comment:
I think it is kind of important that anyone can use some values without having to go through IETF.

Reply:
Agreed. Nothing stops people from having custom keys with their own prefix; however, there's an RFC saying we shouldn't specifically define x-: https://tools.ietf.org/html/rfc6648